google-datacatalog-connectors-commons/src/google/datacatalog_connectors/commons/datacatalog_facade.py

```diff
@@ -55,6 +55,11 @@ def create_entry(self, entry_group_name, entry_id, entry):
             self.__log_entry_operation('was not created',
                                        entry_name=entry_name)
             logging.warning('Error: %s', e)
+        except exceptions.FailedPrecondition as e:
```

Comment:
Nitpick: WDYT about handling the exceptions in alphabetical order to improve readability?